src/components/actions/tests/ActionBar.spec.tsx

```diff
@@ -164,5 +164,16 @@ describe('Actions', () => {
         });
       });
     });
+
+    describe('loading behavior', () => {
+      it('should not have the mod-deactivate-pointer class if the action bar is not loading', () => {
+        expect(actionBar.find('div').first().hasClass('mod-deactivate-pointer')).toBe(false);
```

Comment:
here we should explicitly set the isLoading to false to avoid confusion if we change the default props